### ISSUE TITLE
Session Expire Automatically

### ISSUE THREAD

**Issue by jemspatel** [brave/browser-laptop#2709]
**Did you search for similar issues before submitting this one?**

**Describe the issue you encountered:**
When i have login in Facebook or Gmail (i have tested with these two site) and after some time session expired automatically

```
[12761:0725/135713:ERROR:sandbox_origin_database.cc(198)] SandboxOriginDatabase failed at: Init@../../storage/browser/fileapi/sandbox_origin_database.cc:97 with error: IO error: /home/user/.config/brave/File System/Origins/LOCK: No further details. (ChromeMethodBFE: 15::LockFile::1)
[12761:0725/135713:WARNING:sandbox_origin_database.cc(109)] Attempting to repair SandboxOriginDatabase.
[12761:0725/135714:ERROR:sandbox_origin_database.cc(198)] SandboxOriginDatabase failed at: Init@../../storage/browser/fileapi/sandbox_origin_database.cc:97 with error: IO error: /home/user/.config/brave/File System/Origins/LOCK: No further details. (ChromeMethodBFE: 15::LockFile::1)
[12761:0725/135714:WARNING:sandbox_origin_database.cc(139)] Failed to repair SandboxOriginDatabase.
```

**Expected behavior:**

- Platform (Win7, 8, 10? macOS? Linux distro?):
Win7,  8, 10,  Linux
- Brave Version:
0.11.1
- Steps to reproduce:
    1. Login in gmail / facebook
    2. After some time session expire automatically

- Screenshot if needed:

- Any related issues:

**Comment by bbondy** [comment]
Do you have the option to clear cookies on shutdown on? If so this is already fixed in 0.11.2 but you should keep it off for now until then.

**Comment by jemspatel** [comment]
thanks....
